tests/checkpoint_timestamp/run.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+#
+# Copyright 2019 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+run_sql 'DROP DATABASE IF EXISTS cpts'
+rm -f "$TEST_DIR"/cpts.pb*
+
+export GO_FAILPOINTS="github.com/pingcap/tidb-lightning/lightning/SetTaskID=return(1234567890);github.com/pingcap/tidb-lightning/lightning/restore/FailIfImportedChunk=return(28)"
```

Comment:
`tests/checkpoint_timestamp/data/cpts.cpts.1.sql` is always in one chunk, I think we should split into two chunks to verify the correctness